Makefile

```diff
@@ -58,6 +58,9 @@ ifeq ($(OPERATING_SYSTEM),Windows)
 	choco install wget
 endif
 	wget -O cmake/CPM.cmake https://github.com/TheLartians/CPM.cmake/releases/latest/download/CPM.cmake
+	wget -O sample-data.zip https://github.com/microsoft/electionguard/releases/download/v0.95.0/sample-data.zip
+	unzip sample-data.zip
```

Comment:
you might want to add a -o to this to allow the files to overwrite a previous download.  This is what is cause the build issue